metrics-exporter/base/deploymentconfig.yaml

```diff
@@ -168,21 +168,6 @@ spec:
                 configMapKeyRef:
                   name: prometheus
                   key: instance-metrics-exporter-frontend
-            - name: WORKFLOW_METRICS_BACKEND_PROMETHEUS_INSTANCE
```

Comment:
could you remove that form the configmap in core too?

Reply:
Done :)